address.config.js

```diff
@@ -1,3 +1,5 @@
+const PONTUSX_WALLETS = require('./pontusxWallets.json')
```

Comment:
issue(blocking): We are not referencing any wallets, but addresses. I know that we used to name the property "verifiedWallets" (which is incorrect) but we should be precise with our wording here or might confuse other developers. 

We should update it to something like `const PONTUSX_ADDRESSES = require('./pontusxAddresses.json')`. And accordingly all references to the old property name must be updated as well.

Reply:
Agreed, just didnt see it in scope initially. If it's blocking I'll update, should be quick enough